docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -462,6 +511,35 @@ After training, the model must be destroyed with `ml_train_model_destroy()`.
 `ml_train_model_add_layer()`, `ml_train_set_optimizer()`, and `ml_train_set_dataset()` transfers ownership to the model.
 `layers`, `optimizers` and `dataset` that belongs to the `model` are also deleted.
 
+## Using trained model for the inference
+
+Trained model can be used to inference with [Machine Learning Inference API](machine-learning-inference.md).
+
+Please be sure that the ini points to the correct weight file.
+
+For example, to use with single API(since Tizen 6.5):
```

Comment:
Confusion 

To use ... What ???

Is it ?

For example, since Tizen 6.5, you can use the trained model with single API as follows:

Reply:
Changed as suggested. `For example, since Tizen 6.5, you can use the trained model with single API as follows:`